jcl/src/java.base/share/classes/module-info.java.extra

```diff
@@ -69,8 +69,7 @@ opens jdk.internal.access to
 /*[ENDIF] JAVA_SPEC_VERSION >= 17 */
 exports jdk.internal.misc to
     openj9.criu;
-exports openj9.internal.criu to
-    openj9.criu;
+exports openj9.internal.criu;
```

Comment:
why is this changed? `openj9.internal.criu` should remain restricted to `openj9.criu`

Reply:
Restored `openj9.internal.criu` exports to `openj9.criu` only.